packages/devtools_app/lib/src/screens/memory/shared/heap/model.dart

```diff
@@ -271,7 +275,16 @@ class SnapshotTaker {
   Future<AdaptedHeapData?> take() async {
     final snapshot = await snapshotMemory();
     if (snapshot == null) return null;
-    return AdaptedHeapData.fromHeapSnapshot(snapshot);
+    late final AdaptedHeapData result;
+    ga.timeSync(
+      memory,
```

Comment:
use `MemoryScreen.id` instead

Reply:
oh, nevermind - I see now that this is what `memory` is equal to. Can you instead add a prefix for the constants.dart import?

`import '../../../../analytics/constants.dart' as analytics_constants;`

This is what we do throughout devtools and it helps to make it more clear where these constants are coming from.